content/kotsadm/installing/airgap-packages.md

```diff
@@ -8,6 +8,49 @@ draft: false
 
 When installing an application from an airgap package, the container images and application manifests are provided by the application vendor in an archive that can be used to deliver the artifacts into the cluster. This feature is only available for licenses that have the airgap feature enabled.
```

Comment:
Unrelated to this PR, but we could remove that second line. We're talking from the perspective of the user already having the airgap bundle, so I think we can omit this line. 

```suggestion
When installing an application from an airgap package, the container images and application manifests are provided by the application vendor in an archive that can be used to deliver the artifacts into the cluster. 
```